src/dialog/marisa.c

```diff
@@ -73,8 +138,54 @@ DIALOG_TASK(marisa, Stage3PreBoss) {
 
 	ACTOR_LEFT(marisa);
 	ACTOR_RIGHT(wriggle);
+	HIDE(wriggle);
+	FACE(marisa, normal);
+	MSG(marisa, "Everyone’s more reckless today. What gives?");
 
 	EVENT(boss_appears);
+	MSG_UNSKIPPABLE(wriggle, 180, "Wait until you see their leader!");
+	SHOW(wriggle);
+
+	MSG(marisa, "Yeah, I’m gonna have to ask their leader a bunch of questions for sure.");
+
+	MSG(wriggle, "Feel free to go ahead. I’m listening.");
+
+	MSG(marisa, "Oh, you’re her messenger?");
+	FACE(marisa, happy);
+	MSG(marisa, "Mind passin’ somethin’ along?");
+
+	TITLE(wriggle, "Wriggle Nightbug", "Insect Rights Activist");
+	FACE(wriggle, outraged);
+	MSG(wriggle, "Um, OBVIOUSLY, I’m not the messenger!");
+	FACE(wriggle, proud);
+	MSG(wriggle, "Yeah, I’m the great leader of Insectkind!");
+
+	FACE(marisa, smug);
+	MSG(marisa, "Great leader? What, did the insects unionize or somethin’?");
+
+	FACE(wriggle, outraged);
+	MSG(wriggle, "No! I told you, I’m the leader! The mastermind!");
+	FACE(wriggle, calm);
+	MSG(wriggle, "So, what’s your question?");
+
+	FACE(marisa, normal);
+	MSG(marisa, "Hmm, guess I forgot.");
+	MSG(marisa, "What’s your deal, anyways?");
+
+	MSG(wriggle, "We insects have three demands:");
+	MSG(wriggle, "First: say nay to insect spray!");
+	MSG(wriggle, "Second: swat the swatters!");
+	MSG(wriggle, "Third: the right to bite!");
+	MSG(wriggle, "Fourth—");
+
+	FACE(marisa, unamused);
+	MSG(marisa, "Okay okay, this is great and all, but I’ve honestly got more important stuff to do.");
+	MSG(marisa, "… and y’don’t got a clue about what’s goin’ on, do ya?");
```

Comment:
> y’don’t

I know it's Marisa, but maybe this contraction is a little too much? It took me a moment to decipher; other non-native speakers may struggle too.